README.md

```diff
@@ -2,198 +2,66 @@
-```shell
-python ./backend/node/generate_nodes.py
-BACKEND_ORG=owkin BACKEND_DEFAULT_PORT=8000 ./manage.py init_nodes ./backend/node/nodes/owkinMSP.json --settings=backend.settings.dev
-BACKEND_ORG=chu-nantes BACKEND_DEFAULT_PORT=8001 ./manage.py init_nodes ./backend/node/nodes/chu-nantesMSP.json --settings=backend.settings.dev
-```
-
-## Create a default user
-
-A django admin command is available for registering a user:
-```shell
-./manage.py add_user $USERNAME $PASSWORD
-```
-
-Create these users with:
-
-```shell
-BACKEND_ORG=owkin ./backend/manage.py add_user substra 'p@$swr0d44' --settings=backend.settings.dev
-BACKEND_ORG=chu-nantes ./backend/manage.py add_user substra 'p@$swr0d44' --settings=backend.settings.dev
-```
-
-## Test with unit and functional tests
-
-```shell
-    make test
-    make coverage  # For shell report
-    coverage html  # For HTML report
-```
-
-## Test by creating a traintuple
-
-###### With fixtures
-
-You can test your environment by creating a traintuple:
-```shell
-curl -H "Accept: text/html;version=0.0, */*;version=0.0" -H "Content-Type: application/json" -d '{"algo_key":"da58a7a2-9b54-9f2f-e5f0-09fb51cce6b2","model_key":"","train_data_sample_keys":["62fb3263-208d-62c7-235a-046ee1d80e25","42303efa-6630-15e7-2915-9833a12ffb51"]}' -X POST http://localhost:8001/traintuple/?format=json
-```
-It will try to create a traintuple with creator: chu-nantes (localhost:8001).
-The chu-nantes celery worker will try to add the traintuple to the ledger.
-You can check your traintuple has been corectly added by visiting `http://localhost:8000/traintuple/` or `http://localhost:8001/traintuple/`
-As the trainDataSample passed are also created by chu-nantes, the chu-nantes celery worker will try to update the traintuple status to `training` and save some data if needed.
-You can check `http://localhost:8000/traintuple/` or `http://localhost:8001/traintuple/` to check if the status of your newly created traintuple is set to `training` after 10 sec (celery worker periodic task period).
-You can also check a new objective has been added in `medias/chu-nantes` with a `metrics.py` file but not `description.md` file.
-
-## Launching with docker
-
-As for hlf-k8s, you can launch all the services in docker containers.
-
-First, Make sure you've generated some nodes artifacts:
-```bash
-$> python ./backend/node/generate_nodes.py
-```
-
-Then, build the images:
-```bash
-$> sh build-docker-images.sh
-```
-
-Then, go to the `docker` dir and run `start.py` (`-d` means `dev` settings):
-```bash
-$> python start.py -d --no-backup
-```
+On deployment, several user accounts are created (for [./values/backend-org-1.yaml](org1) and [./values/backend-org-2.yaml](org2)).
 
-Check your services are correctly started with `docker ps -a`.
+The sample credentials for org1 are:
+- user: `node-1`
+- pass: `p@$swr0d44`
 
-## Expiry token period
+You can use them to access the exposed API at http://substra-backend.node-1.com/
```

Comment:
I think this would require to edit the hosts file on the local machine with cluster ip, it could be useful to add a note about this here, like `echo "$(minikube ip) substra-backend.node-1.com substra-frontend.node-1.com substra-backend.node-2.com substra-frontend.node-2.com" | sudo tee -a /etc/hosts`

Reply:
I did not want to repeat the [substra doc](https://doc.substra.ai/setup/local_install_skaffold.html#network) but indeed a link here would make things clearer 👍

Reply:
Yes, great!